app/templates/edit_section.html

```diff
@@ -31,28 +31,19 @@
   %}
     {% include "toolkit/page-heading.html" %}
   {% endwith %}
+
   {% if errors %}
-    <div class="validation-masthead">
-      <h2 class="validation-masthead-heading">
```

Comment:
The toolkit template replaces this `<h2>` heading with an `<h1>`. We end up with two h1s on the page. This seems... bad? And also... happening in a lot of places?

Reply:
Ooh, good spot.  Not sure of the best way out here...

Reply:
**TL;DR** I think this is legit as it is.

Actually, I remember there was a long and involved discussion about this with Tom Byers when he was here.  We ended up agreeing that two H1's on the page are the best thing in this case (yep, honestly...).  I *think* the reasoning went something like this: for accessibility purposes it is generally good practice to have only one H1 on a page.  But in the case where there's an error on the page that you need to fix before you can continue then that is the most important thing you need to know at that point in time, and so is deserving of an H1.

I think this template pattern was only built for G-Cloud 9 submissions, and @samuelhwilliams and @galund might remember the discussion? (Or I might have just dreamed it up).

Reply:
Not sure I recognise this template, so might have been George.

Reply:
Looking at the commit history, the swap to H1 was in line with guidance from GOV.UK elements: https://github.com/alphagov/digitalmarketplace-frontend-toolkit/commit/07f13f8fb6a963f7e87357f71b94e990f40d944c#diff-4388d17aabba821f10d7ca1a64c1e797

Reply:
Although the GOV.UK elements page seems to have downgraded it to an H2 again these days.  I don't think fixing this is part of this bugfix though.

Reply:
Drawing attention to the validation makes sense I suppose. I guess as long as it doesn't confuse the styling or any JS then it's not a blocker (I certainly don't want to make an old discussion any longer / more involved than it has been!)

Reply:
We try to keep in step with GOV.UK thinking on most things like this I think, but it's a moving target and without anyone dedicated to the frontend it's hard for us to keep up. (Changes are the sort of thing one might pick up in the weekly frontend meeting if we had anyone who attended it.)

There probably is a piece of work to be done in bringing our templates into line with most recent GOV.UK styles. (Which would drop this back to an H2), but I think right now by me isn't the time.

Reply:
Fair enough!